worker-pools.yml

```diff
@@ -716,6 +716,49 @@ pools:
           disks:
             - *persistent-disk-75gb
           machine_type: t2a-standard-16
+  - pool_id: '{pool-group}/b-linux'
```

Comment:
Got a bit confused here how we didn't have `b-linux` already but we do https://github.com/mozilla-releng/fxci-config/blob/91cd614722447a94e7c370fdd44bc3e2c279ff6d/worker-pools.yml#L287

It's fine since the trust domains are disjoint but have we thought about maybe calling this differently (like `b-linux-small`) to avoid confusion with the gecko ones?

Reply:
I guess. Just feels a bit weird to decide for projects but then it's probably just me